app/javascript/src/components/Dashboard/Notes/NotesCard/Cards.jsx

```diff
@@ -1,28 +1,28 @@
 import React from "react";
 
-import { MenuVertical, Clock } from "neetoicons";
-import { Tag, Tooltip, Avatar, Dropdown } from "neetoui";
+import { Clock } from "neetoicons";
+import { Tag, Tooltip, Avatar, Typography } from "neetoui";
 import { Header } from "neetoui/layouts";
 
+import CardsDropdown from "./CardsDropdown";
+import { dateTimeFormatter, relativeCreationTime } from "./utils";
+
 import { USER_AVATAR_URL } from "../constants";
-import { dateTimeFormatter, relativeCreationTime } from "../utils";
 
-const { Menu, MenuItem } = Dropdown;
-const Cards = ({ item }) => (
-  <div className="m-3 w-full  border border-gray-200 p-4 shadow-sm  dark:border-gray-500 dark:bg-gray-600">
+const Cards = ({ note, setShowDeleteAlert, setSelectedNoteId }) => (
```

Comment:
The namespace has a lot of redundancies: `Notes/NotesCard/Cards.jsx`

Go through the following and make necessary changes:
- https://academy.bigbinary.com/learn-react/pages/naming-conventions-in-react
- https://bigbinary.slack.com/archives/C01T5Q4DZAR/p1663934324564009?thread_ts=1663060378.971029&cid=C01T5Q4DZAR

Reply:
Hi. Will the namespace be okay if it goes `Notes/Cards/Cards.jsx` and `Notes/Cards/CardsDropdown.jsx` ? Or is a separate folder not a good practice?

Reply:
@KrishnapriyaSkk https://www.loom.com/share/4951b69878ec46069e035fe0effc166f

Reply:
Yes. Thank you so much for the detailed clarification. I have understood the same.

Reply:
>Also, I couldn't use Dropdown as the component name as it conflicts with the 'Dropdown' component of neetoui. I think I should change the naming to something like 'CardDropdown' to make it specific for one single card.

No. Read: https://academy.bigbinary.com/learn-react/pages/miscellaneous-tips#:~:text=Importing%20under%20an%20alias

Reply:
Okay. Thank you so much for clearing out that confusion for me.